hw/ip_templates/alert_handler/data/alert_handler_testplan.hjson

```diff
@@ -8,7 +8,11 @@
                      "hw/dv/tools/dvsim/testplans/enable_reg_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/shadow_reg_errors_testplan.hjson",
                      "hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson",
-                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson"]
+                     "hw/dv/tools/dvsim/testplans/tl_device_access_types_testplan.hjson",
```

Comment:
If we think the above testplan separate is good, I will create another PR to add the common alert_base_testplan.hjson to all IPs that has alerts.